apps/meteor/app/livechat/server/methods/saveIntegration.ts

```diff
@@ -21,6 +21,10 @@ Meteor.methods({
 			await Settings.updateValueById('Livechat_secret_token', trim(values.Livechat_secret_token));
 		}
 
+		if (typeof values.Livechat_http_timeout === 'number') {
```

Comment:
Instead of manually checking types, you can re-use Meteor's helper method `check` :wink: